src/utils/utils_definitions.hpp

```diff
@@ -184,6 +184,7 @@ enum MagicEffectClasses : uint8_t {
 	CONST_ME_YALAHARIGHOST = 66,
 	CONST_ME_BATS = 67,
 	CONST_ME_SMOKE = 68,
+	CONST_ME_STEAM = 68,
```

Comment:
Is this needed and correct? I don't understand why, this is not duplicating the effect?

Reply:
i checked and for now, its 12 code results in opentibiabr/otservbr-global for CONST_ME_SMOKE, so i can only rename SMOKE to STEAM here and fix 12 areas if thats better option for you.

Reply:
> i checked and for now, its 12 code results in opentibiabr/otservbr-global for CONST_ME_SMOKE, so i can only rename SMOKE to STEAM here and fix 12 areas if thats better option for you.

the question is, there is no reason to change this, once you got a really good reason we can reopen the pull request, thanks for your contributions in otservbrglobal datapack